can_spec_my18.yml

```diff
@@ -531,22 +555,31 @@ buses:
             c_type: int32_t
             signed: false
             unit: Wh
-      DashRequest:
+      DashControls:
```

Comment:
I'd like to bunch up all future, potentially non-controls related (if any), dash requests in one message, so make sure to keep everything as small as can be.